ol/tower/src/proof.rs

```diff
@@ -286,9 +288,13 @@ fn create_fixtures() {
         let mut toml_path = PathBuf::from(save_to);
         toml_path.push("miner.toml");
         let file = fs::File::create(&toml_path);
-        file.unwrap()
-            .write(&toml.as_bytes())
-            .expect("Could not write toml");
+        match file {
+            Ok() => file.write(&toml.as_bytes()),
+            Err(msg) => {
+                println!("Could not write toml\nError {}", msg);
+                std::process::exit(1)
+            }
+        }
     }
```

Comment:
`std::process::exit` is a low level fn, should be used in specific cases.
https://doc.rust-lang.org/std/process/fn.exit.html ->
`
Note that because this function never returns, and that it terminates the process, no destructors on the current stack or any other thread’s stack will be run .... 
`
This means memory may not be cleaned at exit, hence big gateway for security/hacking problems.

Suggested way is to use `expect()` which uses `panic` internally instead of `exit` which is also short for match. So the code below actually is equivalent to two matches. One for file creation and one for write.
``` 
        file.expect("failed to create file")
            .write(&toml.as_bytes())
            .expect("Could not write toml");
```

See this also:
https://doc.rust-lang.org/book/ch09-02-recoverable-errors-with-result.html